content/doc/book/glossary/index.adoc

```diff
@@ -129,15 +133,16 @@ Workspace:: [[workspace]]
     <<pipeline,Pipeline>> run completes unless specific Workspace cleanup policies
     have been put in place on the Jenkins <<master,Master>>.
 
-
-////
-XXX: It's currently unclear to me (rtyler) whether these merit definition
-
-== Project/Pipeline Status
+[build-status]
+== Build Status
 
 Aborted:: [[aborted]]
+    The <<build,Build>> was manually aborted.
```

Comment:
Agreed with @res0nance that "manually" is not the right word in this case.
```suggestion
    The <<build,Build>> was stopped before it reached its expected end.
```

Reply:
You're right. However, `stopped` feels a bit ambiguous here. A failed compilation would also stop the build before it reaches its expected end. How about the following?
```suggestion
    The <<build,Build>> was interrupted before it reaches its expected end. For example, the user has stopped it manually or there's a time-out. 
```